code/game/gamemodes/cult/runes.dm

```diff
@@ -1032,14 +1032,17 @@ var/list/teleport_runes = list()
 	N.health = 20
 	N.mouse_opacity = MOUSE_OPACITY_TRANSPARENT
 	new_human.key = ghost_to_spawn.key
-	SSticker.mode.add_cultist(new_human.mind, 0)
+	SSticker.mode.add_cultist(new_human.mind, FALSE)
+	new_human.mind.special_role = "SPECIAL_ROLE_CULTIST"
```

Comment:
No, defines are basically global variables.

Don't make it a string, remove the quotes.